### PR TITLE
See-similiar-proposal-ui-fixes

### PR DESCRIPTION
**Pull request by VAIBHAV-25**

### CODE REVIEW

File: src/components/Post/Post.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ const Post: FC<IPostProps> = (props) => {
 	const isOffchainPost = !isOnchainPost;
 	const [data, setData] = useState<IPostResponse[]>([]);
 	const [isSimilarLoading, setIsSimilarLoading] = useState<boolean>(false);
+	const emptyStateMsg = 'No Active Proposals';
```
Comment (KanishkaRajputd) [inline src/components/Post/Post.tsx:114]: no need to create this variable use directly

File: src/components/Post/Post.tsx [hunk 2]
```diff
@@ -379,6 +380,7 @@ const Post: FC<IPostProps> = (props) => {
 		}
 
 		if (!isOffChainProposalTypeValid(proposalType)) {
+			console.log(post.description);
```
Comment (KanishkaRajputd) [inline src/components/Post/Post.tsx:383]: remove console

File: src/components/Post/Post.tsx [hunk 3]
```diff
@@ -541,36 +543,36 @@ const Post: FC<IPostProps> = (props) => {
...
+						{isSimilarLoading ? (
 							<>
-								<div className='flex items-center'>
-									<hr className='seperation-border mr-2 flex-grow' />
-									<p className='m-0 -mt-[2px] p-0 text-center text-lightBlue'>Discover similar proposals</p>
-									<hr className='seperation-border ml-2 flex-grow' />
-								</div>
 								<div>
 									<LoadingState />{' '}
 								</div>
```
Comment (KanishkaRajputd) [inline src/components/Post/Post.tsx:555]: remove {''}

**Review by KanishkaRajputd — COMMENTED** [review]
check and resolve all the comments

File: src/components/GovernanceCard.tsx [hunk 4]
```diff
@@ -253,22 +259,23 @@ const GovernanceCard: FC<IGovernanceProps> = (props) => {
...
+									<Markdown
+										className='post-content'
+										md={content}
+									/>
 								</p>
-							)}
+							</h1>
+							<p
+								onClick={() => {
+									setShowMore(!showMore);
```
[the remark this answers is not in the document]
Comment (VAIBHAV-25) [reply src/components/GovernanceCard.tsx:273]: wohi krna hai ma'am see more k click p post open (discussed with rajdeep sir and param)

[the remark this answers is not in the document]
Comment (KanishkaRajputd) [reply src/components/GovernanceCard.tsx:268]: cool

**Pull request merged**